src/main/commandServer/settingsValidator.ts

```diff
@@ -259,7 +263,7 @@ export default class SettingsValidator {
     return false;
   }
 
-  protected checkWSLIntegrations(currentValue: Record<string, boolean>, desiredValue: Record<string, boolean>, errors: string[], fqname: string): boolean {
```

Comment:
This probably needs a comment because it's not clear from the function name what it does. (I realize that that is my fault, since I helped with picking the name…)

Even more my fault: all the other functions are `checkX`, instead of `validateX`…

Reply:
Agreed, best to be consistent here. I added some comments, let me know what you think.